pkg/juicefs/mount/builder/common.go

```diff
@@ -96,10 +96,14 @@ func (r *BaseBuilder) genCommonJuicePod(cnGen func() corev1.Container) *corev1.P
 		},
 	}}
 	pod.Spec.Containers[0].Resources = r.jfsSetting.Resources
+	preStopHookCmd := fmt.Sprintf(
+		"umount %s -l; rmdir %s; exit 0", r.jfsSetting.MountPath, r.jfsSetting.MountPath)
+	if r.jfsSetting.ExtraPreStopHookCmd != "" {
+		preStopHookCmd = fmt.Sprintf("%s; %s", r.jfsSetting.ExtraPreStopHookCmd, preStopHookCmd)
```

Comment:
strip the trailing `;` from `r.jfsSetting.ExtraPreStopHookCmd`, if user happens to write something like `echo 1;`, the resulting double `;;` will cause error